src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/eddo/experimental_detectron_detection_output_kernel_ref.cpp

```diff
@@ -0,0 +1,201 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "experimental_detectron_detection_output_kernel_ref.h"
+
+#include <algorithm>
+#include <string>
+
+#include "kernel_selector_utils.h"
+
+namespace kernel_selector {
+
+ParamsKey ExperimentalDetectronDetectionOutputKernelRef::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::INT32);
+    k.EnableInputDataType(Datatype::INT64);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::INT32);
+    k.EnableOutputDataType(Datatype::INT64);
+    k.EnableInputLayout(DataLayout::bfyx);
+    k.EnableOutputLayout(DataLayout::bfyx);
+    k.EnableBatching();
+    k.EnableDifferentTypes();
+    return k;
+}
+
+KernelsPriority ExperimentalDetectronDetectionOutputKernelRef::GetKernelsPriority(const Params&,
+                                                                                  const optional_params&) const {
+    return DONT_USE_IF_HAVE_SOMETHING_ELSE;
+}
+
+bool ExperimentalDetectronDetectionOutputKernelRef::Validate(const Params& p, const optional_params& o) const {
+    if (p.GetType() != KernelType::EXPERIMENTAL_DETECTRON_DETECTION_OUTPUT ||
+        o.GetType() != KernelType::EXPERIMENTAL_DETECTRON_DETECTION_OUTPUT) {
+        return false;
+    }
+    return true;
+}
+
+constexpr int kBoxesInputIdx = 0;
+constexpr int kDeltasInputIdx = 1;
+constexpr int kScoresInputIdx = 2;
+constexpr int kImInfoInputIdx = 3;
+constexpr int kOutputClassesInputIdx = 4;
+constexpr int kOutputScoresInputIdx = 5;
+
+constexpr int kRefinedBoxesBufferIdx = 0;
+constexpr int kRefinedBoxAreasBufferIdx = 1;
+constexpr int kRefinedScoresBufferIdx = 2;
+constexpr int kScoreClassIndexBufferIdx = 3;
+constexpr int kDetectionCountBufferIdx = 4;
+
+constexpr int kBufferCount = 5;
+
+constexpr int kOutputIdx = 0;
+
+constexpr int kRoiCountScalarIdx = 0;
```

Comment:
Build error happened in build_linux_android_x64. Please fix it.
![image](https://user-images.githubusercontent.com/1720339/174957873-393569d9-d8f8-491d-93d5-7acfe1206ed9.png)

Reply:
The build error in `build_linux_android_x64` is fixed.